base/commandLine.cpp

```diff
@@ -329,6 +335,7 @@ void registerDefaults() {
 	// number, then skip scanning. -1 = scan always
 	ConfMan.registerDefault("gui_list_max_scan_entries", -1);
 	ConfMan.registerDefault("game", "");
+	ConfMan.registerDefault("md5-path", "scummvm");
```

Comment:
This is useless since we are not using `ConfMan` to store and get the parameters in `processSettings`.
Currently if no `--md5-path` is specified, an empty path is used for the command, which for me results in an error (`FSNode::createReadStream: 'scummvm' is a directory!`) as it tries to apply the command on the current directory.